buildozer/targets/osx.py

```diff
@@ -93,19 +105,21 @@ def build_package(self):
         author = bc.getdefault('app', 'author', '')
 
         self.logger.info('Create {}.app'.format(package_name))
-        cwd = join(self.buildozer.platform_dir, 'kivy-sdk-packager-master', 'osx')
-        osx_deps = open(join(cwd, 'requirements.txt')).read()
+        osx_deps = ( self.kivy_app_dir / "requirements.txt" ).read_text()
```

Comment:
Huh! `read_text()` New one on me. Thanks!

Reply:
and it's `contexted`!